routes/player.route.js

```diff
@@ -3,14 +3,53 @@ const router = express.Router()
 
 const player_controller = require('../controllers/player.controller')
 
-router.get('/hello', player_controller.hello)
+/**
+ * @api {get} /hello/{name} Prints "Hello {name}"
+ * @apiName Hello
+ * @apiParam (Url) {String} name the name to print
+ * @apiSuccess (200) {String} message the hello {name} message
+ */
+router.get('/hello/:name', player_controller.hello)
```

Comment:
my personal pref is remove the whole `controller` file and just put it here but if you wanna seprate them like you already doing, rename all the `under_score_casing` to `camelCasing`